proto/rill/runtime/v1/resources.proto

```diff
@@ -431,12 +432,11 @@ message DashboardSpec {
 message DashboardState {}
 
 message DashboardComponent {
-  string chart = 1;
+  string component = 1;
   uint32 x = 2;
   uint32 y = 3;
   uint32 width = 4;
   uint32 height = 5;
-  string markdown = 6;
   uint32 font_size = 7;
```

Comment:
Should this stay in the `DashboardComponent` or be moved to the `renderer_properties` for the `markdown` renderer? If the latter, it may require a syntax change since currently it's:
```yaml
kind: component
markdown: Hello world
```
not:
```yaml
kind: component
markdown:
  contents: Hello world
  font_size: 12
```

Another idea is to not support a separately defined font size and let people do it within Markdown using for example `<span>` tags?